src/console.py

```diff
@@ -2,13 +2,16 @@
 
 import argparse
```

Comment:
What's the philosophy for the order of imports here?
or more to the point, why is argparse hanging out alone here?